packages/@glimmer/runtime/lib/compiled/opcodes/debugger.ts

```diff
@@ -19,6 +19,22 @@ function debugCallback(context: unknown, get: DebugGet): void {
   debugger;
 }
 
+function templateOnlyDebugCallback(get: DebugGet): void {
+  console.info('Use `get(<@path>)` to debug this template.');
```

Comment:
Is it possible to get all passed arguments, unsing such path notation? (kinda `this.args`)...

Reply:
Ya, we should probably do something like this automatically, but the current plumbing for it doesn't exist (AFAICT). Seems good to do in a follow up...